command/operator_migrate.go

```diff
@@ -339,7 +346,7 @@ func parseStorage(result *migratorConfig, list *ast.ObjectList, name string) err
 
 // dfsScan will invoke cb with every key from source.
 // Keys will be traversed in lexicographic, depth-first order.
-func dfsScan(ctx context.Context, source physical.Backend, cb func(ctx context.Context, path string) error) error {
+func dfsScan(ctx context.Context, source physical.Backend, cb func(ctx context.Context, path string) error, wg sync.WaitGroup) error {
```

Comment:
`wg` needs to be a `*sync.WaitGroup`, otherwise each call will get a copy and the top level `Wait` won't work.